packages/shared/src/modules/users/libs/enums/user-validation-message.enum.ts

```diff
@@ -1,6 +1,8 @@
 const UserValidationMessage = {
 	EMAIL_REQUIRE: "Email is required",
 	EMAIL_WRONG: "Email is wrong",
+	PASSWORD_REQUIRE: "Password is not allowed to be empty",
+	PASSWORD_SHORT: "Password length must be at least 8 characters long",
```

Comment:
```suggestion
	PASSWORD_SHORT: "Password length must be at least ${UserValidationRule.PASSWORD_MINIMUM_LENGTH} characters long",
```

Reply:
Have to change cons to String or have:
  7:54  error  Invalid type "8" of template literal expression  @typescript-eslint/restrict-template-expressions